### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ZuowenWang0000**
added work 'Exploiting Spatial Sparsity for Event Cameras with Visual Transformers'

### CODE REVIEW

**Comment by daydayupzhang** [comment]
你好，我已经收到你的来信。

File: README.md [hunk 1]
```diff
@@ -1618,6 +1618,9 @@ IEEE Int. Conf. Computer Vision (ICCV), 2021.
 - <a name="li21iccv"></a>Li, Y., Zhou, H., Yang, B., Zhang, Y., Cui, Z., Bao, H., Zhang, G.,  
 *[Graph-based Asynchronous Event Processing for Rapid Object Recognition](https://openaccess.thecvf.com/content/ICCV2021/html/Li_Graph-Based_Asynchronous_Event_Processing_for_Rapid_Object_Recognition_ICCV_2021_paper.html)*,  
 IEEE Int. Conf. Computer Vision (ICCV), 2021. [PDF](https://openaccess.thecvf.com/content/ICCV2021/papers/Li_Graph-Based_Asynchronous_Event_Processing_for_Rapid_Object_Recognition_ICCV_2021_paper.pdf), [Suppl.](https://openaccess.thecvf.com/content/ICCV2021/supplemental/Li_Graph-Based_Asynchronous_Event_ICCV_2021_supplemental.pdf)
+- <a name="wang22icip"></a>Wang, Z., Hu, Y., Liu, S.-C,  
+*[Exploiting Spatial Sparsity for Event Cameras with Visual Transformers](https://ieeexplore.ieee.org/document/9897432)*,  
+IEEE Int. Conf. on Image Processing (ICIP), 2022. [PDF](https://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=9897432).
```
Comment (guillermogb) [inline README.md:1623]: The second PDF id a link to a document that is now behind a paywall, if you have it.

**Review by guillermogb — CHANGES_REQUESTED** [review]
thanks

File: README.md (hunk 1 above)
Comment (ZuowenWang0000) [reply README.md]: hi, thanks. I changed the second pdf link to the arxiv version. Does this work?

**Pull request merged**